judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/transfer/MappedTransferPrimitivesTest.java

```diff
@@ -0,0 +1,1036 @@
+        assertEquals(LocalTime.parse("23:59:59"), transferWithRequiredPrimitiveDefaults.getTimeAttr());
+        // There is no way to define default value in JSL for binary
+        // assertEquals("test.txt", entityWithDefaults.getBinaryAttr().get().getFileName());
+        assertEquals(Enum.EnumA, transferWithRequiredPrimitiveDefaults.getEnumAttr());
+    }
+
+    /**
+     * The transfer object mapped entity with required default primitive expression. The test checks the required default expression are present in the transfer object too.
+     *
+     * @prerequisites The model runtime is empty. It means that the database of the application has to be empty before this test starts running.
+     *
+     * @type Behaviour
+     *
+     * @others Implement this test case in the *judo-runtime-core-jsl-itest* module.
+     *
+     * @jslModel MappedTransferPrimitives.jsl
+     *
+     * @scenario
+     *
+     *  Create one TransferWithPrimitiveDefaultExpressions instance.
+     *
+     *  Check all fields contain the default primitives expression.
+     *
+     */
+    @Test
+    @TestCase("TransferCreationWithPrimitiveDefaultExpressions")
+    @Disabled("https://blackbelt.atlassian.net/browse/JNG-4815")
+    @Requirement(reqs = {
+            "REQ-TYPE-001",
+            "REQ-TYPE-002",
+            "REQ-TYPE-004",
+            "REQ-TYPE-005",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-TYPE-009",
+            "REQ-ENT-001",
+            "REQ-ENT-002",
+            "REQ-MDL-001",
+            "REQ-MDL-002",
+            "REQ-MDL-003",
+            "REQ-SRV-002",
+            "REQ-EXPR-001",
+    })
+    public void testTransferCreationWithPrimitiveDefaultExpressions() {
+        TransferWithPrimitiveDefaultExpressions transferWithPrimitiveDefaultExpressions = transferWithPrimitiveDefaultExpressionsDao.create(TransferWithPrimitiveDefaultExpressions.builder().build());
+
+        List<TransferWithPrimitiveDefaultExpressions> list = transferWithPrimitiveDefaultExpressionsDao.query().execute();
+
+        assertEquals(1, list.size());
+
+        assertEquals(Optional.of(1), transferWithPrimitiveDefaultExpressions.getIntegerAttr());
+        assertEquals(Optional.of(2.9), transferWithPrimitiveDefaultExpressions.getScaledAttr());
+        assertEquals(Optional.of("true"), transferWithPrimitiveDefaultExpressions.getStringAttr());
+        assertEquals(Optional.of("+36-1-123-123"), transferWithPrimitiveDefaultExpressions.getRegexAttr());
+        assertEquals(Optional.of(true), transferWithPrimitiveDefaultExpressions.getBoolAttr());
+        assertEquals(Optional.of(LocalDate.now()), transferWithPrimitiveDefaultExpressions.getDateAttr());
+        assertEquals(OffsetDateTime.now().toString().substring(1, 10),
+                transferWithPrimitiveDefaultExpressions.getTimestampAttr().get().toString().substring(1, 10));
```

Comment:
This "feels" wrong:D

I suggest using constant values for default expressions if we are not explicitly testing temporal operations.

Reply:
Add comment to use constains, when the default values work

Reply:
[6f85e57](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/165/commits/6f85e57884996176f05ef4554ae4669dee9e50a2)